pkgs/development/tools/depot_tools/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, stdenvNoCC, targetPlatform, fetchgit }:
+with lib;
+let
+  rev = "25cf78395cd77e11b13c1bd26124e0a586c19166";
+
+  cipd-platform-arch = if targetPlatform.isi686 then "386"
+  else if targetPlatform.isS390 && targetPlatform.is64bit then "s390x"
+  else if targetPlatform.isx86_64 then "amd64"
+  else (let
+    str = "${targetPlatform.parsed.cpu.family}${if targetPlatform.is64bit then "64" else "32"}";
+    ed = if (targetPlatform.parsed.cpu.family == "mips" || targetPlatform.isPower64) && targetPlatform.isLittleEndian then
+      "le"
+    else "";
+  in "${str}${ed}");
+
+  cipd-platform-kernel = if targetPlatform.isDarwin then
+    "mac"
+  else if targetPlatform.isLinux or targetPlatform.isWindows then
+    targetPlatform.parsed.kernel.name
+  else throw "Unsupported kernel ${targetPlatform.parsed.kernel.name}";
+in stdenvNoCC.mkDerivation rec {
+  pname = "depot_tools";
+  version = "git+${rev}";
+
+  src = fetchgit {
+    url = "https://chromium.googlesource.com/chromium/tools/depot_tools.git";
+    inherit rev;
+    sha256 = "sha256-Qn0rqX2+wYpbyfwYzeaFsbsLvuGV6+S9GWrH3EqaHmU=";
+  };
+
+  passthru.cipd = {
+    # Get from "cipd_client_version" in depot_tools
+    version = "git_revision:89ada246fcbf10f330011e4991d017332af2365b";
+
+    # Get from "cipd_client_version.digests" in depot_tools
+    hashes = builtins.listToAttrs (map (line:
+      let
+        segments = builtins.split " +" line;
+      in { name = builtins.head segments; value = lib.last segments; })
+    (lib.filter (line: !(hasPrefix "#" line || line == "")) (splitString "\n" (readFile ./cipd_client_version.digests))));
```

Comment:
use the file `${depot_tools.outPath}/cipd_client_version.digests`

Reply:
That will not work.